src/js/controller/instream-adapter.js

```diff
@@ -137,7 +137,7 @@ const InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
             }
         });
         _model.set('instream', _adProgram);
-        _adProgram.model.set('state', STATE_PLAYING);
+        _adProgram.model.set('state', STATE_LOADING);
```

Comment:
If anything `_adProgram.model` 'state' should be set to equal the player state, and then you can force an adPlay if that is the expectation when an ad is already playing.